src/util.cpp

```diff
@@ -594,7 +594,8 @@ CAboutDlg::CAboutDlg ( QWidget* parent ) : CBaseDlg ( parent )
 
     // libraries used by this compilation
     txvLibraries->setText ( tr ( "This app uses the following libraries, resources or code snippets:" ) + "<p>" +
-                            tr ( "Qt cross-platform application framework" ) + QString ( " %1" ).arg ( QT_VERSION_STR ) +
+                            tr ( "Qt cross-platform application framework" ) + QString ( " %1 " ).arg ( QT_VERSION_STR ) + tr ( "(build)" ) +
```

Comment:
Can we think of another name then running? Maybe runtime?

Running is a bit ambiguous

Reply:
Happy with "runtime", unless anyone else has a better suggestion?

Reply:
Or "current"? Or "now"?

Reply:
I'd prefer "runtime" (as that's the general term -- runtime libraries).  End user is only expected to use this to cut-n-paste into a bug report, not understand the terminology or implications, after all.

Reply:
What happens with a static build -- does it correctly report the same for both values?

Reply:
> What happens with a static build -- does it correctly report the same for both values?

Yes, it should, as it uses standard methods to retrieve the two values.

Although I think our binaries for Mac and Windows are not static builds, but they dynamically link to private libraries provided by the Jamulus installer. I'm not sure how to make a static build to test.

> (Ideally, the message could indicate whether the binary was running against dynamic libraries or was a static build. If dynamic libraries, we'd also want to know where they were, of course -- are they ones our installer installed, or are they supplied by the platform?)

I don't know how this would be achieved, nor whether it is really worth working out how?

Reply:
> > What happens with a static build -- does it correctly report the same for both values?
> 
> Yes, it should, as it uses standard methods to retrieve the two values.

OK

> Although I think our binaries for Mac and Windows are not static builds, but they dynamically link to private libraries provided by the Jamulus installer. I'm not sure how to make a static build to test.

You can download the source for Qt and create the object files and then do a static build in Qt Creator.  (I used to do this when brewing Jamulus before the current build system was in place.  With the dynamic build, using a locally built version isn't as easy -- much easier to push to a branch and run the GitHub builder - but much slower.)

> (Ideally, the message could indicate whether the binary was running against dynamic libraries or was a static build. If dynamic libraries, we'd also want to know where they were, of course -- are they ones our installer installed, or are they supplied by the platform?)
> 
> I don't know how this would be achieved, nor whether it is really worth working out how?

If it was trivial, it would be worth it as it would be useful for diagnostics, which is all any of this is _useful_ for.  (Quite apart from any licence compliance issues...)  But I've no idea how to do it, either 😄 .

Reply:
> But I've no idea how to do it, either 😄 .

OK, so are you able to approve it as it is?